modules/analyse/src/main/ui/AnalyseUi.scala

```diff
@@ -31,6 +32,8 @@ final class AnalyseUi(helpers: Helpers)(externalEngineEndpoint: String):
         description = "Analyse chess positions and variations on an interactive chess board"
       )
       .flag(_.zoom):
+        val positionNum =
+          pov.game.variant.chess960.so(Chess960.positionNumber(initalFen | chess.format.Fen.initial))
```

Comment:
it would be better to get the positionNum as an argument, rather than the initialFen, only to later recompute the positionNum